core/src/test/java/org/mskcc/cbio/portal/web_api/TestGetProfileData.java

```diff
@@ -74,6 +74,7 @@ public void setUp() throws DaoException {
 		newGeneticProfile.setStableId("study_tcga_pub_test");
 		newGeneticProfile.setProfileName("Barry CNA Results");
 		newGeneticProfile.setDatatype("test");
+		newGeneticProfile.setReferenceGenomeId(1);
```

Comment:
here

Reply:
reference_id assigned not auto generated. will not change.